src/components/reference/_triage-states.mdx

```diff
@@ -18,5 +18,8 @@ Findings can also be **removed**. Semgrep considers a finding removed if it is n
 Your removed findings do not count toward the fix rate or the number of findings. The removed findings also do not appear in Semgrep AppSec Platform.
 
 :::info Findings triaged across refs
```

Comment:
I wonder if we want this to be an anchor rather than callout? And do we want to update the header?

Options:

- Findings triaged across refs
- Triage behavior across refs or branches
- Triaging findings across refs or branches
- How Semgrep triages findings across refs or branches